src/MarkdownTextInput.web.tsx

```diff
@@ -328,15 +329,14 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
         if (!divRef.current || !(e.target instanceof HTMLElement)) {
           return;
         }
-
-        if (compositionRef.current) {
+        const nativeEvent = e.nativeEvent as MarkdownNativeEvent;
+        if (compositionRef.current && (nativeEvent.inputType !== 'insertCompositionText' || nativeEvent.data !== '*')) {
```

Comment:
I doubt it will function properly if the single character entered is `*` [here](https://github.com/Expensify/App/issues/40799#top)